src/Quote.js

```diff
@@ -0,0 +1,92 @@
+import React, { Fragment } from "react";
+import { New } from "./New.js";
+
+export function Quote({
+  color, 
+  loading, 
+  error,
+  customError, 
+  data,
+  handleClick
+}) {
+  return(
+    <Fragment>
+
+      { loading && 
+        <div id="loading-box">
+          <img 
+              class="icon"
+              src="images/loader.gif" 
+              alt="Eye glasses' arms opening and closing with eyes roving and blinking" 
+            />
+          <p class="primary-text">Quote incoming!</p>
+        </div>
+      }
+
+      { error && 
+        <div>
+          <img 
+              class="icon"
+              src="images/error.png" 
+              alt="Triangle with '!' inside to indicate error" 
+            />
+          <p class="primary-text">{error}</p>
+          <p className="custom-message">{customError}</p>
+          
+          <New 
+            handleClick={handleClick}
+            color={color}
+          />
+
+        </div> 
+      }
+
+      { data &&
+        <Fragment>
+
+          <p className="custom-message">(These quotes are not endorsed by the creator of this app.)</p>
+
+          <div 
+            id="quote"
+            style={{backgroundColor: color}}
+          >
+            <p id="text" class="primary-text">{data.content}</p>
+            <p id="author">{"—" + data.originator.name}</p>
+          </div>
+
+          <span id="tweet-container">
+            <img 
+              id="twitter-logo"
+              src="images/twitter.svg" 
+              alt="Twitter logo" 
+            />
+            <a 
+              id="tweet-quote" 
+              href={"https://twitter.com/intent/tweet?text=“" + data.content + "” —"+data.originator.name + ",&via=iggldee"} 
+              target="_blank" 
+              rel="noopener noreferrer" 
+              OnClick="window.open(this.href)" 
+              title="Share on Twitter"
+            >
+              Tweet it<i>!</i>
+            </a>
+          </span>
+
+          <New 
```

Comment:
Second place `New` component is called. Here, it needs slightly different styling in a couple of ways than the other place it is called above, so it is passed the `color` and `className` props.